docs/src/basic-tutorial.md

```diff
@@ -302,19 +301,19 @@ You can switch between those workflows however you please.
 ```@example
 using CairoMakie
 
-f = Figure()
-ax = Axis(f[1, 1])
+fig = Figure()
+ax = Axis(fig[1, 1])
 hm = heatmap!(ax, randn(20, 20))
-Colorbar(f[1, 2], hm, width = 20)
-f
+Colorbar(fig[1, 2], hm, width = 20)
+fig
 ```
 
-## Passing attributes to implicit Figure and Axis
+## Passing attributes to Figure and Axis
 
-For one-off plots, it's convenient to set a few axis or figure settings directly with the plotting command.
-You can do this only with the plotting functions without `!` like `lines` or `scatter`, because those always create a new axis, and can create a new figure if they are not plotting into an existing one. This is explained further under [Plot Method Signatures](@ref).
+For one-off plots, it can be convenient to set axis or figure settings directly with the plotting command.
+You can do this using the plotting functions without the `!` suffix, like `lines` or `scatter`, because thoese always create a new axis and also create a new figure if they are not plotting onto an existing one. This is explained further under [Plot Method Signatures](@ref).
```

Comment:
```suggestion
You can do this using the plotting functions without the `!` suffix, like `lines` or `scatter`, because these always create a new axis and also create a new figure if they are not plotting onto an existing one. This is explained further under [Plot Method Signatures](@ref).
```

Reply:
Thanks!